SU-SOE/stanford_soe_helper
# 7.x 2.x alpha3 tagging and Turn off "Disable SQL Rewrite" in views after core patch

# READY FOR REVIEW

# Summary
- Turn off "Disable SQL Rewrite" in views
- Update all version numbers in .info files to 7.x-1.0-beta3 (WM)
- Update CHANGELOG.txt with a human-readable version of what’s been changed since 7.x-1.0-beta2 (WM)


# Needed By (04/25/2018)

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Make sure all the .info files and CHANGELOG.txt files are correct with the 7.x-1.0-beta3 update
- Get the latest patched core: https://code.stanford.edu/sws/drupal-sites/commits/stable20170525-soe-patch
- After core patch, this PR changes $handler->display->display_options['query']['options']['disable_sql_rewrite'] = TRUE; to $handler->display->display_options['query']['options']['disable_sql_rewrite'] = FALSE; 
- This turns off the option in the view for the query to "disable sql rewrite" 
- The collection view should show up for users not logged in without having that query option selected.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2850

## More Information

## Folks to notify
@minorwm
